docs/csharp/whats-new/csharp-version-history.md

```diff
@@ -168,4 +168,86 @@ All of these features offer cool new capabilities for developers and the opportu
+  - A combination of syntax improvements that enable working with value types using reference semantics.
+- [Non-trailing named arguments](./csharp-7-2.md#non-trailing-named-arguments)
+  - Named arguments can be followed by positional arguments.
+- [Leading underscores in numeric literals](./csharp-7-2.md#leading-underscores-in-numeric-literals)
+  - Numeric literals can now have leading underscores before any printed digits.
+- [`private protected` access modifier](./csharp-7-2.md#private-protected-access-modifier)
+  - The `private protected` access modifier enables access for derived classes in the same assembly.
+- [Conditional `ref` expressions](./csharp-7-2.md#conditional-ref-expressions)
+  - The result of a conditional expression (`?:`) can now be a reference.
+
+## C# version 7.3
+
+There are two main themes to the C# 7.3 release. One theme provides features that enable safe code to be as performant as unsafe code. The second theme provides incremental improvements to existing features. In addition, new compiler options were added in this release.
+
+The following new features support the theme of better performance for safe code:
+
+- You can access fixed fields without pinning.
+- You can reassign `ref` local variables.
+- You can use initializers on `stackalloc` arrays.
+- You can use `fixed` statements with any type that supports a pattern.
+- You can use additional generic constraints.
+
+The following enhancements were made to existing features:
+
+- You can test `==` and `!=` with tuple types.
+- You can use expression variables in more locations.
+- You may attach attributes to the backing field of auto-implemented properties.
+- Method resolution when arguments differ by `in` has been improved.
+- Overload resolution now has fewer ambiguous cases.
+
+The new compiler options are:
+
+- `-publicsign` to enable Open Source Software (OSS) signing of assemblies.
+- `-pathmap` to provide a mapping for source directories.
+
+## C# version 8.0
+
+C# 8.0 is the first major C# release that specifically targets .NET Core. Some features rely on new CLR capabilities, others on library types added only in .NET Core. C# 8.0 adds the following features and enhancements to the C# language:
+
+- [Readonly members](./csharp-8.md#readonly-members)
+- [Default interface methods](./csharp-8.md#default-interface-methods)
+- [Pattern matching enhancements](./csharp-8.md#more-patterns-in-more-places):
+  - [Switch expressions](./csharp-8.md#switch-expressions)
+  - [Property patterns](./csharp-8.md#property-patterns)
+  - [Tuple patterns](./csharp-8.md#tuple-patterns)
+  - [Positional patterns](./csharp-8.md#positional-patterns)
+- [Using declarations](./csharp-8.md#using-declarations)
+- [Static local functions](./csharp-8.md#static-local-functions)
+- [Disposable ref structs](./csharp-8.md#disposable-ref-structs)
+- [Nullable reference types](./csharp-8.md#nullable-reference-types)
```

Comment:
Now that we have an article dedicated to this, should we link directly to that article?